wallet/bindings/java/lib/native/src/lib.rs

```diff
@@ -247,16 +254,15 @@ unsafe fn event_handle(clazz: GlobalRef, callback_ref: GlobalRef, event: Event)
 
     // Make the Jni object to send back
     let event = jni::sys::jvalue {
-        l: make_jni_string(&env, ev_ser),
+        l: make_jni_string(&mut env, ev_ser),
     };
 
     // Get a ref back to the callback we call on the java side
-    let cb = JValue::Object(callback_ref.as_obj()).to_jni();
-    let clazz = JClass::from_raw(*clazz.as_obj());
+    let cb = JValue::Object(callback_ref.as_obj()).as_jni();
 
     // Call NativeApi, METHOD_CACHE assumed initialised if we received a VM
     let res = env.call_static_method_unchecked(
-        clazz,
+        &clazz,
```

Comment:
Just curious, is `clazz` a common thing in Java?

Reply:
yes, as class is a keyword, you write clazz